entproto/service.go

```diff
@@ -30,6 +30,9 @@ const (
 	// MaxPageSize is the maximum page size that can be returned by a List call. Requesting page sizes larger than
 	// this value will return, at most, MaxPageSize entries.
 	MaxPageSize = 1000
+	// MaxBatchCreateSize is the maximum number of entries that can be created by a single BatchCreate call. Requests
+	// exceeding this batch size will return an error.
+	MaxBatchCreateSize = 1000
```

Comment:
Wasn't sure if we wanted granular control (setting for each batch operation) here or one batch size limit for all operations.

Reply:
This is a good start, we can add this in a small PR later.